src/main/java/org/sunbird/workflow/service/impl/WorkflowServiceImpl.java

```diff
@@ -1370,7 +1370,7 @@ public Response getUserProfileApprovalRequest(String rootOrg, String org, Search
 				totalRequestCount = applicationIdsPage.getTotalElements();
 			}
 			if (StringUtil.isNotBlank(criteria.getQuery()) && criteria.getServiceName().equals(Constants.PROFILE_SERVICE_NAME)) {
-				if (StringUtil.isBlank(rootOrgId) && (criteria.getRequestType().equals(Constants.GROUP_CHANGE) || criteria.getRequestType().equals(Constants.DESIGNATION_CHANGE))) {
+				if (StringUtil.isBlank(rootOrgId) && (criteria.getRequestType().contains(Constants.GROUP_CHANGE) || criteria.getRequestType().contains(Constants.DESIGNATION_CHANGE))) {
```

Comment:
you must always check against constant... 

Constants.GROUP_CHANGE.contains(criteria.getRequestType())